SS2-Project/Assets/Starstorm2/Modules/Pickups/Items/Tier3/ErraticGadget.cs

```diff
@@ -56,7 +56,7 @@ public override void Initialize()
         // figure out if a lightningorb "ends", then figure out how many objects it bounced to, then spawn a gadgetlightningorb with the same stats that bounces to that many targets
         private void LightningOrb_OnArrival(On.RoR2.Orbs.LightningOrb.orig_OnArrival orig, LightningOrb self)
         {
-            if(!self.target) // i dont like this... but im also lazy
+            if(&& !self.target) // i dont like this... but im also lazy
```

Comment:
this logical AND, i'm 99.9% sure its a compilation error.

Reply:
whoops